functional-tests/network-deployment/relay_check_test.go

```diff
@@ -303,12 +303,8 @@ func networkBootstrapPeers(network string) []string {
 	// Currently all bootstrap addresses are relay peers
 
 	switch network {
-	case "nightly":
-		return fixtures.DevnetNightlyBootstrapAddrs
-	case "staging":
-		return fixtures.DevnetStagingBootstrapAddrs
-	case "user":
-		return fixtures.DevnetUserBootstrapAddrs
+	case "interop":
+		return networks.InteropNet.Bootstrap.Addresses
```

Comment:
@travisperson we should consider whether these network deployment tests are worth keeping in this form - I'd appreciate your advice (to follow-up).

Reply:
This test can definitely go. We are not using relays.